integration/configs/transcription_test.config.json

```diff
@@ -10,13 +10,10 @@
         "noOfThreads": 1,
         "testImpl": "TranscriptionTest.js",
         "payload": {
-          "url": "http://localhost:8080/",
+          "url": "http://localhost:8080/?region=us-east-1",
+          "region": "us-east-1",
```

Comment:
Don't think you need this if you specify it in the URL already.

Reply:
The metrics get the region string from the payload rather than the query parameter.

Reply:
Then do you need the query parameter?

Reply:
Updated to only use the payload parameter and wired it up through the test.